pylint/checkers/base/comparison_checker.py

```diff
@@ -15,6 +15,7 @@
 COMPARISON_OPERATORS = frozenset(("==", "!=", "<", ">", "<=", ">="))
 TYPECHECK_COMPARISON_OPERATORS = frozenset(("is", "is not", "==", "!="))
 TYPE_QNAME = "builtins.type"
+SINGLETON_VALUES = {True, False, None}
```

Comment:
I don't like this pattern, it adds coupling, and we never know if those constant can be removed even if the corresponding code do not exists anymore. Unless we use it twice or thrice it should not be a constant.